Entity/Task.php

```diff
@@ -97,9 +99,9 @@ public function getHandlerClass()
      *
      * @param string $handlerClass
      *
-     * @return $this
+     * @return Task
      */
-    public function setHandlerClass($handlerClass)
+    public function setHandlerClass(string $handlerClass): self
```

Comment:
the phpdoc of most of the methods of this class doesnt really add any value and could be removed